host/host.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/flynn/flynn/pkg/cluster"
 	"github.com/flynn/flynn/pkg/shutdown"
 	"github.com/flynn/flynn/pkg/version"
+
+	_ "github.com/flynn/flynn/Godeps/_workspace/src/github.com/inconshreveable/gonative"
```

Comment:
It was getting deleted when I ran `godeps` because there's no import reference in the project. This probably isn't the best spot for it though.